VERSION

```diff
@@ -1 +1 @@
-1.0.0
```

Comment:
ha.conf is common file

all other file are only for v1
consul_watcher_failback.json 
database.json
decision_monitor_conf.json
decision_monitor_conf_vm.json
rules_engine_schema_jbod.json
...ules_engine_schema_with_crossconnect.jsoon
...s_engine_schema_without_crossconnect.json

Reply:
Done.